meshtastic/mesh_interface_runtime/flows.py

```
@@ -0,0 +1,561 @@
+                    )
+            else:
+                telemetry_dict = google.protobuf.json_format.MessageToDict(telemetry)
+                for key, value in telemetry_dict.items():
+                    if key != "time":
+                        if isinstance(value, dict):
+                            _emit_response_summary(f"{key}:")
+                            for sub_key, sub_value in value.items():
+                                _emit_response_summary(f"  {sub_key}: {sub_value}")
+                        else:
+                            _emit_response_summary(f"{key}: {value}")
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)